app/utils.py

```diff
@@ -61,9 +61,29 @@ def get_campaign_site_link(request, display_region, requested_link):
                 link = 'https://www.ons.gov.uk/aboutus/contactus/surveyenquiries'
         elif requested_link == 'privacy':
             if display_region == 'cy':
-                link = base_cy + '/privacy-and-data-protection/'
+                link = base_cy + '/privacy-and-data-protection'
             else:
-                link = base_en + '/privacy-and-data-protection/'
+                link = base_en + '/privacy-and-data-protection'
```

Comment:
I think this would need to be reverted, but also I don't think this function is ever actually called for this privacy link any more?

Reply:
Yeah, it's never used with 'surveys-home' and 'privacy' as argument. I've been told to leave it as it is, but I can remove it.